test/resources/valid-trusts-registration-api-5mld-nontaxable.json

```diff
@@ -0,0 +1,332 @@
+{
+  "correspondence": {
+    "abroadIndicator": false,
+    "name": "1234567890 QwErTyUiOp ,.(/)&'- name",
+    "address": {
+      "line1": "1234567890 QwErTyUiOp ,.(/)&'- name",
+      "line2": "1234567890 QwErTyUiOp ,.(/)&'- name",
+      "postCode": "SR2 7UY",
+      "country": "GB"
+    },
+    "phoneNumber": "+123456789012345678",
+    "welsh": false,
+    "braille": false
+  },
+  "declaration": {
+    "name": {
+      "firstName": "1234567890 QwErTyUiOp ,.(\/)&'- name",
+      "lastName": "1234567890 QwErTyUiOp ,.(\/)&'- name"
+    },
+    "address": {
+      "line1": "DECLARED WITH NINO",
+      "line2": "DECLARED WITH NINO",
+      "postCode": "XX1 1XX",
+      "country": "GB"
+    }
+  },
+  "trust": {
+    "details": {
+      "startDate": "2019-04-06",
+      "trustTaxable": false,
+      "expressTrust": true,
+      "trustUKResident": true,
+      "trustUKProperty": true,
+      "lawCountry": "DE",
+      "administrationCountry": "DE",
```

Comment:
Minor comment that the test data here doesn't align to what would be expected for a non-taxable trust.